### ISSUE TITLE
Readme Cleanup

### ISSUE THREAD

**Issue by alxjrvs** [TheGnarCo/gnarails#135]
The README still references `bundle exec` terms, instead of the popular rails 5.0 Spring-based `bin/rake` `bin/rails` etc. May be worthwhile to scan through the Readme and see if there are any other updates to be made.

**Comment by kevin-j-m** [comment]
To be clear, you're talking about the templated README that's generated after using gnarails, and not the README for gnarails directly, correct?

**Comment by alxjrvs** [comment]
Yes, exactly. The Generated Rails Readme, not the Readme for Gnar-rails itself.
